lib/browser/api/session.ts

```diff
@@ -1,11 +1,37 @@
 import { fetchWithSession } from '@electron/internal/browser/api/net-fetch';
 import { net } from 'electron/main';
 const { fromPartition, fromPath, Session } = process._linkedBinding('electron_browser_session');
+const { isDisplayMediaSystemPickerAvailable } = process._linkedBinding('electron_browser_desktop_capturer');
```

Comment:
I wonder if it'd be worth exposing this as part of the public API as well. I'm fine punting on it for now since there's time pressure to release this, but dropping a comment in case we want to think about a follow-up change.

Reply:
I'd be open to exploring that - originally, this was actually exposed as part of the public API, and we decided to move this under the hood to simply the logic for devs. However, if we want to expose it publicly in the future, that should be fairly trivial to do

Reply:
I'd be open to exploring that - originally, this was actually exposed as part of the public API, and we decided to move this under the hood to simply the logic for devs. However, if we want to expose it publicly in the future, that should be fairly trivial to do